taichi/codegen/cuda/codegen_cuda.cpp

```diff
@@ -667,10 +667,6 @@ class TaskCodeGenCUDA : public TaskCodeGenLLVM {
         llvm_val[stmt] = create_call("__nv_powf", {lhs, rhs});
       } else if (ret_type->is_primitive(PrimitiveTypeID::f64)) {
         llvm_val[stmt] = create_call("__nv_pow", {lhs, rhs});
-      } else if (ret_type->is_primitive(PrimitiveTypeID::i32)) {
```

Comment:
nit: can we add a comment here to redirect readers to the demote pass? :P I'm afraid people might accidentally miss that and add these back in the future lol

Reply:
SG!